3d_detection_tracking/run.sh

```diff
@@ -3,8 +3,6 @@
 source activate apolloscape
```

Comment:
Also this is for local environment, rm them. Please check other code for similiar issues